Library/Formula/syncthing.rb

```diff
@@ -2,6 +2,10 @@ class Syncthing < Formula
   homepage "http://syncthing.net"
   url "https://github.com/syncthing/syncthing.git", :tag => "v0.10.24"
 
+  head do
+    url "https://github.com/syncthing/syncthing.git"
```

Comment:
Since this doesn't require any other dependencies or patches, you can remove the ` head ` block and just stick a ` head "https://example.git" ` line underneath ` url ` above.